Lection 3/task 3.1/ReportApp/Services/Activity/ActivityReportDataService.cs

```diff
@@ -62,53 +59,62 @@ public void FillReportDataFromModel(XLTemplate template, ActivityReportConfigura
             KeyValuePairs.Add("Works At", r => r.GeneratedByAdmin.City);
         }
 
+        int initialLastRow = configuration.LastRow;
```

Comment:
Are we using it?

Reply:
Yes, "_templateService.DrawBorders(worksheet, configuration, "Activity", lastDataColumn, initialLastRow);" line 103